web/Spinner.qml

```diff
@@ -16,52 +18,52 @@ WebItem {
 
 	onRunningChanged: {
 		spinnerItem.transform.rotate += 360;
-		spinnerItem.squize = ++spinnerItem.squize % 5;
+		spinnerItem.squize = ++spinnerItem.squize % (spinnerItem.width * 5 / 80)
```

Comment:
does not look right to me, why do you get modulo of width * 5/ 80?

Reply:
because we want everything to be proportional to the item dimensions. Note that when width = 80, the result is exactly the same we had before this change

Reply:
I looked at the code and what I just wrote it makes no sense. Sorry about that, I'll fix it.